addons/mail_group/data/mail_template_data.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record id="mail_template_guidelines" model="mail.template">
-        <field name="name">Mail Group: Send Guidelines</field>
+        <field name="name">Mailing list guidelines</field>
```

Comment:
This is misleading, as mailing lists exist in mass mailing application (and are a different model with a different usage than mail groups). Mail Group is the real naming we use, so I would not change it (or change it to something else than mailing list, but we did not find anything clearer).